include/MainWindow.h

```diff
@@ -21,29 +22,30 @@ class MainWindow : public QMainWindow
     Q_OBJECT
 
 public:
-    MainWindow(QWidget *parent = nullptr);
+    MainWindow(QWidget* parent = nullptr);
     ~MainWindow();
 
-    QStatusBar *getStatusBar() const {
+    QStatusBar* getStatusBar() const {
         return statusBar;
     }
 
     const int statusBarShortMessageDuration = 7000;
 
 private:
-	// UI components
-    Dockable *objectTreeWidgetDockable;
-    Dockable *objectPropertiesDockable;
-    Dockable *toolboxDockable;
-    QStatusBar *statusBar;
+    HelpWidget* helpWidgetInstance;
```

Comment:
No need to name it to Instance. So remove it and keep it like this `HelpWidget* helpWidget;`